Mathlib/Tactic/ClearValue.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2022 Evan Lohn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Evan Lohn, Mario Carneiro
+-/
+import Lean
+
+namespace Mathlib.Tactic.ClearValue
+open Lean Meta Elab Tactic Term
+
+syntax (name := clearValue) "clear_value" (ppSpace colGt ident)* : tactic
+
+
+def getNameAndLDecl [Monad m] [MonadError m] (ctx : LocalContext) (stx : Syntax) :
```

Comment:
Do we want dot notation here, so we can write `ctx.getNameAndLDecl`?

Reply:
I'd discourage introducing new abbreviations, i.e. don't abbreviate LocalDecl to LDecl unless this is already widespread.